packages/core/src/Stores/client-store.js

```diff
@@ -576,6 +576,27 @@ export default class ClientStore extends BaseStore {
         return this.residence === 'gb';
     }
 
+    @computed
+    get country_standpoint() {
+        const result = {
+            is_uk: this.is_uk,
+            is_france: this.residence === 'fr',
+            is_belgium: this.residence === 'be',
```

Comment:
I think it is better to follow one naming convention for all countries. all should be abbreviated or all should be full name:

```js
            is_uk: this.is_uk,
            is_fr: this.residence === 'fr',
            is_be: this.residence === 'be',
```

or 

```
            is_united_kingdom: this.is_uk,
            is_france: this.residence === 'fr',
            is_belgium: this.residence === 'be',
```

But now the `UK` is abbreviated and other ones are full. (I suggest using abbr for all)